packages/react/documentation/coding-conventions.md

```diff
@@ -0,0 +1,31 @@
+# Coding Conventions
+
+## Barrel files
+
+In our React package we use barrel files (`index.ts` in the root and in every component folder) to simplify the import of React components.
+Barrel files allow a consumer to do this:
+
+`import { Heading, Link, Paragraph } from '@amsterdam/design-system-react'`
+
+Instead of this:
+
+```js
+import Heading from "@amsterdam/design-system-react/dist/Heading/Heading";
+import Link from "@amsterdam/design-system-react/dist/Link/Link";
+import Paragraph from "@amsterdam/design-system-react/dist/Paragraph/Paragraph";
+```
+
+However, barrel files have 2 potential pitfalls:
+
+- Without proper treeshaking, importing from a barrel file can mean that a consumer imports the entire library, instead of just the components they need. To avoid this, the consumer needs to set up treeshaking in their project (TODO: add documentation on this? Or at least describe both methods of importing?)
+- When a component imports from a sibling component, using a barrel file for this import can lead to cyclical dependencies.
```

Comment:
```suggestion
However, barrel files have two potential pitfalls:

1. Without proper treeshaking, importing from a barrel file can mean that a consumer imports the entire library, instead of just the components they need. To avoid this, the consumer needs to set up treeshaking in their project (TODO: add documentation on this? Or at least describe both methods of importing?)
2. If a component imports from a sibling component, using a barrel file can lead to cyclical dependencies.
```

Reply:
[Numbers are more easy to understand as numbers vs text](https://www.amsterdam.nl/schrijfwijzer/heldere-taal-basis-onze-huisstijl/tekstonderdelen-heldere-taal/getallen-bedragen-breuken-percentages/).

Having an ordered list makes sense though, I'll change it!

Reply:
I updated the suggestion to return the 2.